src/dbg/taskthread.cpp

```diff
@@ -0,0 +1,95 @@
+#include <Windows.h>
```

Comment:
Okay so it seems like you define WINVER somewhere in the command line or something. Possibly a global setting in Visual Studio, maybe an environment variable? But yeah in that case the WINVER define should probably be added in the project files.